### ISSUE TITLE
GUI won't load after install (CLI OK)

### ISSUE THREAD

**Issue by mcgrovia** [MicrosoftTranslator/DocumentTranslation#103]
I've installed the GUI after downgrading to .NET 6.0.0 (i couldn't get --roll-forward to work to run the existing 6.0.10). It appears to install correctly, and the command line works to translate a file. I run DocumentTranslation.GUI.exe (incl. as admin), it 'thinks' then nothing. No evidence of it being open in taskmgr. I've also tried opening the GUI from the CLI, no errors or feedback of any kind. I haven't been able to use a folder as a source (will post this separately) so I'm hoping the GUI will work. 

Win 11 Pro, .NET Desktop Runtime 6.0.0, .NET Runtime 6.0.10 (should I roll this back to match?)

**Comment by chriswendt1** [comment]
Yes, you should update to the current .Net 6. Unpredictable what happens with an old version.,
What made you downgrade?

**Comment by mcgrovia** [comment]
Thank you Chris. I downgraded following an error on installation of Document Translation saying I needed to install .NET (it was already installed).   I followed the links to install/update but kept getting the same error (even though it was installed, and I restarted each attempt).  The error message referenced 6.0.0 (making me think that is what it was searching for). Once i downgraded to 6.0.0 (from 6.0.10) the installation completed successfully (although without a functioning GUI). I have now brought it back to 6.0.10 and the GUI loads. Can't explain it - I've tried to replicate the initial installation error with no luck. Happy to have it working, thank you for the prompt reply. I'm sorry I cant provide more detail on the initial cause.
